versions.yaml

```diff
@@ -40,4 +40,5 @@ module-sets:
       - go.opentelemetry.io/otel/schema
 excluded-modules:
   - go.opentelemetry.io/otel/internal/tools
+  - go.opentelemetry.io/otel/sdk/log/xlog
```

Comment:
I think that relying on git hashes would be very annoying for the users
 It would be hard for them to keep `sdk/log` and `sdk/log/xlog` in sync.
1. We are making commits very frequently
2. It would be hard to figure it out which version of `xlog` is compatible with `log` with we do any breaking change.

On the other side, we can wait for making a release until people actually complain.

Thoughts?

Reply:
For comparison, the collector keeps x packages within the normal release process.

I have doubts about the idea, but if we want to allow folks to be able to get very fresh code, we could setup auto nightly/weekly releases for the x packages. Then, they wouldn't rely on commits, but would remain on the latest.